src/v3/src/transformer/layout/email/emailChallengeTransformer.ts

```diff
@@ -127,11 +128,13 @@ export const transformEmailChallenge: IdxStepTransformer = ({ transaction, formB
           titleElement,
           informationalText,
           showCodeStepperButton,
-        ],
+        ].map((ele: UISchemaElement) => ({ ...ele, viewIndex: 0 })),
```

Comment:
This adds a `viewIndex` value to the element so we can determine which view we are on when adding describedby values.